vivarium/core/store.py

```diff
@@ -362,7 +362,10 @@ def set_path(self, path, value):
                 self.insert({
                     'processes': value.generate_processes({'name': final}),
                     'topology': value.generate_topology({'name': final}),
-                    'initial_state': value.initial_state()})
+                    # If a process's initial_state is expected, this will need to use inverse_topology
+                    # 'initial_state': value.initial_state()
```

Comment:
Can we just remove this instead? We have the git history, so I don't see any benefit to leaving the commented-out code

Reply:
sure.